pages/signupform.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+    Button,
+    Box,
+    Center,
+    Text,
+    Input,
+    FormControl,
+} from "@chakra-ui/react";
+
+/**
+ * This is the page that a user will use to either login or register
+ * to the SDC platform as a parent of volunteer
+ */
+export default function Signupform(): JSX.Element {
+    return (
+        <Center h="500px">
+            <Box width="700px">
+                <Center>
+                    <Text fontWeight="700" fontSize="36px">
+                        Sign Up
+                    </Text>
+                </Center>
+                <Center>
+                    <Text fontWeight="400" fontSize="18px" mt="18px">
+                        I am a ...
+                    </Text>
+                </Center>
+                <Center>
+                    <Button
+                        backgroundColor="transparent"
+                        borderColor="#8D8D8D"
+                        width="366px"
+                        height="54px"
+                        fontSize="16px"
```

Comment:
@mahadzaryab1 Would it be useful to specify certain font size and font weights as constants so that it's consistent throughout?

Reply:
I think font sizes don't to be constants because they will likely vary throughout the app

Reply:
@mahadzaryab1 Aren't there colours that will be used more frequently in the design?